bin/propolis-server/src/lib/initializer.rs

```diff
@@ -282,7 +282,7 @@ impl<'a> MachineInitializer<'a> {
         uuid: uuid::Uuid,
     ) -> Result<(), anyhow::Error> {
         if let Some(ref spec) = self.spec.devices.qemu_pvpanic {
-            if spec.enable_isa {
+            if spec.enable_mmio {
```

Comment:
I think "ISA" was more accurate when it comes to naming, even if the linux driver calls it MMIO.

Reply:
will change it back!